docker-compose.yml

```diff
@@ -24,9 +24,19 @@ services:
             - '/usr/src/app/node_modules'
         ports:
             - 3008:8080
-        depends_on: 
+        depends_on:
             - unitednepali-server
-    
+
+    unitednepali-documentation:
```

Comment:
Should include a conatiner name, it'll be easier to understand when debugging, because if you don't give a name docker will assign one, which can be hard to keep track of.

Reply:
Ohhh that's what that does! Should I add the `env_file: docker/.env` as well?

Reply:
nope, `docker/.env` are just to store environment variables to pass on to the container.